Doc/library/pickle.rst

```diff
@@ -494,7 +494,7 @@ What can be pickled and unpickled?
 
 The following types can be pickled:
 
-* ``None``, ``True``, and ``False``;
+* built-in constants (``None``, ``True``, ``False``, ``Ellipsis``, and ``NotImplemented``);
```

Comment:
Wrap to 79 characters per earlier suggestion (some lines in this file don't, but it remains best practice).
```suggestion
* built-in constants (``None``, ``True``, ``False``, ``Ellipsis``, and
  ``NotImplemented``);
```